api/v1alpha/framework/imagerunner.schema.json

```diff
@@ -69,6 +69,10 @@
         },
         "timeout": {
           "$ref": "../subschema/common.schema.json#/definitions/timeout"
+        },
+        "workload": {
+          "description": "Sets the kind of workload that is being executed",
+          "type": "string"
```

Comment:
We should set this as an enum 🙏 Considering that this is a fairly important field and there aren't that many workload types to choose from. That'll also make it easier to help validate within the IDE.

Reply:
Moved it to an enum with `webdriver` and `other` values.
I had to use `other` (at least, non-blank), an empty value is currently as `webdriver` by the backend.

Reply:
I know you guys don't like the generated code but a model with enums is generated [here](https://github.com/saucelabs/hostedrunner-api/blob/main/api/v1alpha1/client.gen.go#L21-L22)

Reply:
That has been updated accordingly !